resources/assets/less/status.less

```diff
@@ -0,0 +1,349 @@
+/**
+*    Copyright 2015 ppy Pty. Ltd.
+*
+*    This file is part of osu!web. osu!web is distributed with the hope of
+*    attracting more community contributions to the core ecosystem of osu!.
+*
+*    osu!web is free software: you can redistribute it and/or modify
+*    it under the terms of the Affero GNU General Public License version 3
+*    as published by the Free Software Foundation.
+*
+*    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+*    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+*    See the GNU Affero General Public License for more details.
+*
+*    You should have received a copy of the GNU Affero General Public License
+*    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+*
+*/
+// main: compiled.less
+@import "app";
+
+@smallScreen: 0px;
+@medScreen: 610px;
+@bigScreen: 1220px;
+
+#statusPage {
+  .incidents__list {
+    .incident__state {
+      width: 34px;
+      height: 16px;
+      content: ' ';
+      float: left;
+      background: red;
+      border-radius: 4px;
+      margin: 10px;
+      margin-left: 20px;
+      margin-top: 4px;
+
+      @media (min-width: @smallScreen) and (max-width: @bigScreen) {
+        margin-left: 10px;
+      }
+
+      &.incident__state--resolved {
+        background: #A2CC00 !important;

```

Comment:
![](https://puu.sh/njl7C/f4b4818547.png)

https://css-tricks.com/when-using-important-is-the-right-choice/